### PR TITLE
Api latest

### PR DESCRIPTION
**Pull request by adydychk**

### CODE REVIEW

**Comment by onabiull** [comment]
I would prefer to squash all "fix" commits to meaningful ones. But still LGTM.

**Comment by adydychk** [comment]
@onabiull Be sure, I will squash it. We squash ALL commit (as in devs repositories). This branch was used for developers purposes so I have so many commits.  
P.S.  
If you are interested I can explained you face-to-face how we deploy our infrastructure using git :)

**Comment by adydychk** [comment]
Actual documentation: https://github.com/Intel-Media-SDK/infrastructure/pull/16

File: build_scripts/build_runner.py [hunk 1]
```diff
@@ -720,7 +720,7 @@ def main():
                         choices=['release', 'debug'],
                         help='Type of build')
     parser.add_argument('-p', "--product-type", default='linux',
-                        choices=['linux', 'embedded', 'open_source', 'windows'],
```
Comment (eponomar) [inline build_scripts/build_runner.py]: Don't remove 'open_source'. It's needed for closed open source builds)

Comment (adydychk) [reply build_scripts/build_runner.py]: Ok. Strange that it was deleted...

Comment (adydychk) [reply build_scripts/build_runner.py]: Approve pls

**Pull request merged**